### PR TITLE
[WIP] Mongodb setup

### PR DESCRIPTION
**Pull request by ff-l**

### CODE REVIEW

File: src/content_api/details_utils.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+                    'licence_overview', 'licence_short_description', 'logo', 'metadata', 'more_information',
+                    'need_to_know',
+                    'other_ways_to_apply', 'summary', 'ways_to_respond', 'what_you_need_to_know', 'will_continue_on',
+                    'parts',
+                    'collection_groups']
+
+
+def is_json(raw_text):
+    try:
+        json_normalize(raw_text).columns.tolist()
```
Comment (suganya-s) [inline src/content_api/details_utils.py]: Is this quicker than trying `json.loads(raw_text)`?/

Comment (ff-l) [reply src/content_api/details_utils.py]: I don't think it makes a difference.

**Comment by suganya-s** [comment]
All looks good, although the structures you're dealing with seem pretty weird, did you do this through inspecting lots of data or finding some documentation somewhere?

`content_purpose_document_supertype` is [deprecated](https://docs.publishing.service.gov.uk/document-types/content_purpose_document_supertype.html) so you might as well not keep that column in your notebooks.

**Comment by ff-l** [comment]
@suganya-s no documentation as far as I know, trial/error + experience with similar nesting. Will exclude column + refactor the json check

**Comment by suganya-s** [comment]
Ah I wish there was some documentation 😂 looks like you're found all the things we need atm though, and looks pretty extensible for other usecases

**Review by suganya-s — APPROVED** [review]

**Pull request merged**